index.rst

```diff
@@ -312,6 +312,16 @@ and the supported values associated with them:
      - Must be specified in the preamble.  Set to ``true`` to tell the core
        not to check any higher directory for EditorConfig settings for on the
        current filename.  The value is case-insensitive.
+   * - ``project_icon.<THEME>``
```

Comment:
`project_icon` (without a `.<THEME>`) appears to be unspecified.  Is that intended?